gallery/src/components/NxCollapsibleMultiSelect/NxCollapsibleMultiSelectExample.tsx

```diff
@@ -45,7 +52,9 @@ const NxCollapsibleMultiSelectExample = () => {
   const [filter, setFilter] = useState('');
 
   function filterPredicate(option: NxCollapsibleMultiSelectOption) {
-    return includes(toLower(filter), toLower(option.name));
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const name = typeof option.name === 'string' ? option.name : (option.name as any).props.children[1].props.children;
```

Comment:
Let's make a follow-up ticket to improve this. It should be possible to add some type magic to both remove the need to cast to string in the other examples, and make a better way to check the name in the JSX here.  But that doesn't need to be done today in RSC 13.0

Reply:
> Let's make a follow-up ticket to improve this. It should be possible to add some type magic to both remove the need to cast to string in the other examples, and make a better way to check the name in the JSX here. But that doesn't need to be done today in RSC 13.0

Agreed, this is hardly ideal. Ticket made: https://sonatype.atlassian.net/browse/RSC-1736